packages/gafl-webapp-service/src/pages/order-complete/order-complete/__tests__/order-complete.spec.js

```diff
@@ -103,6 +198,97 @@ describe('The order completion handler', () => {
     )
   })
 
+  it.each`
+    desc                                                                | licenceFor | postal   | method      | expected
+    ${'Postal licence for you with none digital confirmation'}          | ${true}    | ${true}  | ${'Letter'} | ${postalYouNoneDigitalConf}
+    ${'Postal licence for someone else with none digital confirmation'} | ${false}   | ${true}  | ${'Letter'} | ${postalElseNoneDigitalConf}
+    ${'Postal licence for you with digital confirmation'}               | ${true}    | ${true}  | ${'Text'}   | ${postalYouDigitalConf}
+    ${'Postal licence for someone else with digital confirmation'}      | ${false}   | ${true}  | ${'Text'}   | ${postalElseDigitalConf}
+    ${'Digital licence for you'}                                        | ${true}    | ${false} | ${'Text'}   | ${youDigital}
+    ${'Digital licence for someone else'}                               | ${false}   | ${false} | ${'Text'}   | ${elseDigital}
+  `('$desc', async ({ desc, licenceFor, postal, method, expected }) => {
+    const permission = getSamplePermission({ isLicenceForYou: licenceFor, postalFulfilment: postal, preferredMethodOfConfirmation: method })
+    const { content } = await getData(getSampleRequest({ permission }))
+    expect(content).toEqual(expected)
```

Comment:
If you used a snapshot test, you could dispense with the array of expected variables up above:
```
    const permission = getSamplePermission({ isLicenceForYou: licenceFor, postalFulfilment: postal, preferredMethodOfConfirmation: method })
     const { content } = await getData(getSampleRequest({ permission }))
     expect(content).toMatchSnapshot()
```